code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -44,6 +44,14 @@
 		core = null
 	return ..()
 
+/mob/living/carbon/xenomorph/hivemind/Life() //If we have no weeds within Range 1 we now get booted back to our core.
+	. = ..()
+	var/obj/effect/alien/weeds/W = locate() in range(1, get_turf(src))
+	if(!W)
+		forceMove(get_turf(core))
+		to_chat(src, "<span class='xenonotice'>We lacked weeds to sustain our presence and our consciousness was shunted to our core.")
+		return FALSE
```

Comment:
I'm not particularly happy with this being in `life()` Yes I understand that this is the 'ideal' spot for keeping the hivemind on weeds, but we've already got checks in `Move()` and it's `plant weeds` ability. allowing it to 'stand vigil' on a tile is not the worst thing in the world.

Reply:
Being able to just indefinitely spam weeds and other crap on the same tile, regardless of marine efforts to clear it so long as you don't move is definitely an issue, and should not be possible.

Reply:
because we have the checks in the abilities, said hivemind cannot spam weeds.

```
/datum/action/xeno_action/plant_weeds/slow
	cooldown_timer = 12 SECONDS

/datum/action/xeno_action/plant_weeds/slow/action_activate()
	if(locate(/obj/effect/alien/weeds) in range(1, owner.loc))
		return ..()
	var/mob/living/carbon/xenomorph/hivemind/hiveminde = owner
	hiveminde.forceMove(get_turf(hiveminde.core))
	to_chat(hiveminde, "<span class='xenonotice'>We can't plant a node without weeds nearby, we've been moved back to our core.</span>")
	return fail_activate()
```

We want it moved to the core whenever it tries to do something not on weeds. I don't want this code to be run every life tick.

Reply:
> The idea that hivemind has no impact on the round is laughable. They have more impact than AI, their counterpart, while being significantly harder to destroy and deal with. You don't see anyone saying the AI should have direct offensive abilities?
> 
> There is no counterplay to a hivemind. You can't shoot the eye, chase it off, burn it or corner it. Marines can spend a good 10 minutes constantly clearing weeds from a FOB or Hangar when one wants to be cheeky. Without any risk to themselves they can heal any xeno at any point on the map. A drone or hivelord must physically be there, and often putting themself at risk. They are slower builders but rightly fully so, they can ghost through their own constructs and given enough time make autism inspiring levels of walls in an area, and drop down new walls if a marine destroys one.
> 
> Top this off, the only time the hivemind ever dies is when the hivemind themselves screwed up and put their core in an easy to find spot, usually to close to the FOB/LZ. In most cases, the hivemind survives even when the rest of the hive dies during a hijacking.
> 
> What's your answer to how to marines are supposed to counter a role giving them brain damage during an SD defense when the core literally isn't located on the ship and they can't fly back down to deal with it? There is very good reasons the hivemind has no offensive capabilities and generally good arguments are made for the hivemind to be removed or completely rework to not be an uncounterable ghost griefing marines throughout a round.

I didn't say the Hivemind has **no** impact, my argument is that it isn't worth the precious slot, and it most certainly isn't when the opportunity cost is a Drone or better; there's really no contest. You are greatly overvaluing the omnipresence and the difficulty of killing them. Neither of these things matter so much when you have so little power to influence the battle in comparison means little. Hands down I would prefer a Drone to a Hivemind as it stands.

Brain damage has been removed in exchange for higher Stamina damage. Further, any use of the Core Reposition tips off Marines to the Hivemind's location now.

Further a lot of the abusive oversights in the code RE: Hivemind are being corrected in this PR regarding FOB harassment.

Reply:
This is done.